src/con_duct/__main__.py

```diff
@@ -406,7 +409,8 @@ def dump_json(self) -> str:
         )
 
     def print_summary(self) -> None:
-        print(self.summary_format.format_map(self.execution_summary))
+        human_readable = {k: "unknown" if v is None else v for k, v in self.execution_summary.items()}
+        print(self.summary_format.format_map(human_readable))
```

Comment:
is there a unittest which would test this function that logic of conversion to `human_readable` is correct?

Reply:
there is now ;)